linux/just_files/new_just

```diff
@@ -474,97 +479,117 @@ function write_dockerfile()
 {
   exists "${1}" && return 0
 
-  uwecho   'FROM vsiri/recipe:gosu as gosu
+  uwecho   '#T# Dockers use gosu instead of sudo for smoother operation
+            FROM vsiri/recipe:gosu as gosu
+            #T# Dockers do not have a zombies reape by default, tini does this
             FROM vsiri/recipe:tini as tini
+            #T# Just dockers use vsi common
             FROM vsiri/recipe:vsi as vsi' >> "${1}"
 
   if [ "${USE_PIPENV}" = "1" ]; then
-    uwecho 'FROM vsiri/recipe:pipenv as pipenv
-            ## # Uncomment for GDAL
-            FROM vsiri/recipe:gdal as gdal
+    uwecho '#T# Download and setups up pipenv for you
```

Comment:
while that's true, the pipenv recipe is just a script that downloads and setup pipenv (not a pipenv environment) so that you can call pipend that will then "manages the python virtual env and installs a consistent set of packages". So I was saying what this really does...